src/app/[locale]/globals.css

```
@@ -22,126 +22,243 @@
+  -webkit-transform: translateY(40px);
+  -moz-transform: translateY(40px);
+  -ms-transform: translateY(40px);
+  -o-transform: translateY(40px);
+}
+
+.turnstile-transition.turnstile-animated {
+  animation: turnstileFadeIn 3500ms ease-in forwards;
+  -webkit-animation: turnstileFadeIn 3500ms ease-in forwards;
+}
+
+@keyframes turnstileFadeIn {
```

Stylelint (17.6.0)

[error] Expected keyframe name "turnstileFadeIn" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)